addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8646,8 +8661,10 @@ msgctxt "#19056"
 msgid "New timer"
 msgstr ""
 
+#. Label for a context menu entry to open timer settings dialog
+#: xbmc/pvr/windows/GUIWindowPVRTimers.cpp
 msgctxt "#19057"
-msgid "Edit timer"
+msgid "Edit settings"
```

Comment:
Just "Edit" might also do, as context menu already has "Deactivate" and "Delete". But as it also has "New Timer" and "Find similar" (similar what btw - timers? shows?). "New timer" could IMO be dropped from the context menu as there is a dedicated button on the window